src/perl/Permabit/BlockDevice/VDO.pm

```diff
@@ -600,20 +609,26 @@ sub installModule {
                                   modName         => $moduleName,
                                   modVersion      => $version,
                                   useDistribution => $self->{useDistribution},
+                                  useDnf          => $self->{useLinuxNextModule},
                                  );
```

Comment:
There is no reason to change the flag name here.
The new flag should have a name something like useUpstream or useBuiltin. The kernel module side does not use dnf so it's not a good name for the generic case.